### PR TITLE
[17.0] [MIG] stock_quant_reservation_info:  Migration to 17.0

### PR DESCRIPTION
**Pull request by MeritxellAForgeFlow**
Migration to 17.0 
@ForgeFlow

### CODE REVIEW

File: stock_quant_reservation_info/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Stock Move Reservation Info",
     "summary": "Allows to see the reserved info of Products",
-    "version": "16.0.1.0.1",
+    "version": "17.0.1.0.1",
```
Comment (JordiMForgeFlow) [inline stock_quant_reservation_info/__manifest__.py]: reset to 17.0.1.0.0

File: stock_quant_reservation_info/views/stock_quant.xml
Comment (DavidJForgeFlow) [inline stock_quant_reservation_info/views/stock_quant.xml:1]: In this version there is two views for the quants, one going throught Inventory -> Operations -> Inventory Adjustments (`stock.view_stock_quant_tree_inventory_editable`) and the other one going throught Inventory -> Products -> Products and clicking on the smart button Onhand inside of a product (`stock.view_stock_quant_tree_editable`) make sure the changes of the modules reflect on both views!

**Review by DavidJForgeFlow — CHANGES_REQUESTED** [review]
Works fine! But there is a small detail for this version!

File: stock_quant_reservation_info/views/stock_quant.xml
Comment (AaronHForgeFlow) [reply stock_quant_reservation_info/views/stock_quant.xml:1]: Edit: my previous commit was not correct. The quant list view modified is located in Physical inventory menu.

The problem is the module stock_inventory deletes that menu and I was looking at the wrong place.

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review :+1:

**Review by JoanSForgeFlow — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Functional review

~I miss the button to show reservation info~

EDIT: this review was from the other PR #2208. In this one I see the button but I see the reserved qty 2 times:

image

**Review by LoisRForgeFlow — APPROVED** [review]
Now it LGTM!

**Pull request merged**